### ISSUE TITLE
Add support to Gradle 4.4

### ISSUE THREAD

**Issue by paolorotolo** [prolificinteractive/material-calendarview#748]
Right now it's impossible to compile the project in Android Studio 3.1 with Gradle 4.4.

```Cannot create variant 'android-lint' after configuration ':library:debugRuntimeElements' has been resolved```

The error is known, please check how other developers [fixed it](https://github.com/google/ExoPlayer/issues/4047).

**Comment by paolorotolo** [comment]
The issue seems to be caused by those 2 lines in library's build.gradle:

```
        source = variant.javaCompile.source
        classpath = files(variant.javaCompile.classpath.files, project.android.getBootClasspath())
```

**Comment by paolorotolo** [comment]
@quentin41500 PLEASE MERGE THIS

**Comment by tevjef** [comment]
Thank you for reporting the issue. Resolved in #759.
